test/integration/tasks_test.rb

```diff
@@ -56,7 +56,7 @@ def rake(cmd, options = {})
   test "rails perf:library with bad script" do
     # BUNDLE_GEMFILE="$(pwd)/gemfiles/rails_git.gemfile" bundle exec m test/integration/tasks_test.rb:<linenumber>
 
-    skip unless ENV['USING_RAILS_GIT']
+    skip # unless ENV['USING_RAILS_GIT']
```

Comment:
This test errors out during setup: fd9308a2925e862435859e1803e720e6eebe4bb6 is reported as an invalid ref so the DERAILED_SCRIPT never gets a chance to run